daffodil-cli/src/it/scala/org/apache/daffodil/CLI/Util.scala

```diff
@@ -43,9 +42,8 @@ object Util {
   val binPath = Paths.get(dafRoot, "daffodil-cli", "target", "universal", "stage", "bin", String.format("daffodil%s", (if (isWindows) ".bat" else ""))).toString()
 
   def getExpectedString(filename: String, convertToDos: Boolean = false): String = {
-    val rsrc = Misc.getRequiredResource(outputDir + filename)
-    //val source = scala.io.Source.fromFile(outputDir + filename)
-    val source = scala.io.Source.fromFile(rsrc)
+    val is = getClass.getResourceAsStream(outputDir + filename)
```

Comment:
There is a reason to use the Misc library, which is that you get a diagnostic that lists out the classpath including all the directories, jars, etc. if the thing isn't found. I think it fixes up the typical "no leading slash" problem as well.  I think I wrote the Misc.getRequiredResource when trying to debug classpath issues with include/import.

Reply:
getRequiredResource returned a path inside a jar (e.g. file://path/to/foo.jar!/path/inside/jar.txt) and I could not find a way to easily get the contents of that file. I suspect in the past it found the file on the classpath so didn't need to look inside a jar. I'll reinvestigate this to figure out how to get the contents of a resource inside a jar.

Reply:
val dataURI = Misc.getRequiredResource("iso8859.doc.dat")
    val dataInput = dataURI.toURL.openStream()

Reply:
This fixed it. Thanks!